src/hooks/useReleaseHistory.tsx

```diff
@@ -40,52 +51,34 @@ export function useReleaseHistory(): ReleaseData[] {
   return releaseHistory;
 }
 
-export function getStaticReleaseData(): NodeReleaseData[] {
-  return [
-    {
-      release: 'v10',
-      status: 'Maintenance LTS',
-      codename: 'Dubnium',
-      initialRelease: '2018-04-24',
-      activeLTSStart: '2018-10-30',
-      maintenanceLTSStart: '2020-05-19',
-      endOfLife: '2021-04-30',
-    },
-    {
-      release: 'v12',
-      status: 'Active LTS',
-      codename: 'Erbium',
-      initialRelease: '2019-04-23',
-      activeLTSStart: '2019-10-21',
-      maintenanceLTSStart: '2020-10-20',
-      endOfLife: '2022-04-30',
-    },
-    {
-      release: 'v14',
-      status: 'Active LTS',
-      codename: 'Fermium',
-      initialRelease: '2020-04-21',
-      activeLTSStart: '2020-10-27',
-      maintenanceLTSStart: '2021-10-19',
-      endOfLife: '2023-04-30',
-    },
-    {
-      release: 'v15',
-      status: 'Curent',
-      codename: '',
-      initialRelease: '2020-10-20',
-      activeLTSStart: '',
-      maintenanceLTSStart: '2021-04-01',
-      endOfLife: '2021-06-01',
-    },
-    {
-      release: 'v16',
-      status: 'Pending',
-      codename: '',
-      initialRelease: '2021-04-20',
-      activeLTSStart: '2021-10-26',
-      maintenanceLTSStart: '2022-10-18',
-      endOfLife: '2024-04-30',
-    },
-  ];
+export function useReleaseData(): NodeReleaseData[] {
+  const releaseDataURL =
+    'https://raw.githubusercontent.com/nodejs/Release/main/schedule.json';
+  const [releaseData, setReleaseData] = useState<NodeReleaseData[]>([]);
+  useEffect((): void => {
+    const fetchData = async (): Promise<void> => {
+      const result = await fetch(releaseDataURL).then(
```

Comment:
Nit: Mixing async/await with then() looks weird for me.

Reply:
This was previously implemented like this actually. Will change that